src/components/Search/SearchRouter/SearchRouter.tsx

```diff
@@ -56,6 +58,49 @@ function SearchRouter() {
         return OptionsListUtils.getSearchOptions(options, '', betas ?? []);
     }, [areOptionsInitialized, betas, options]);
 
+    /**
+     * Builds a suffix tree and returns a function to search in it.
+     */
+    const findInSearchTree = useMemo(() => {
+        const fastSearch = FastSearch.createFastSearch([
+            {
+                data: searchOptions.personalDetails,
+                toSearchableString: (option) => {
+                    const displayName = option.participantsList?.[0]?.displayName ?? '';
+                    return [option.login ?? '', option.login !== displayName ? displayName : ''].join();
```

Comment:
Does it matter that we will have a string with like `,something` if the login is an empty string or just `,` if both are empty? Not sure if it can happen in practice, but maybe we should not include any empty strings at all.

Reply:
empty strings won't be searchable. No characters would be added for that to the search tree, thus they'd never be retrievable